addons/website/static/src/xml/website.pageProperties.xml

```diff
@@ -173,6 +173,7 @@
                             </t>
                             <t t-else="">
                                 <t t-set='groups_tooltip'>More than one group has been set on the view.</t>
+                                <!-- FIXME is it normal to target a specific view id here ? -->
                                 <a class="show_group_id btn btn-link mx-auto" href="/web#model=ir.ui.view&amp;id=681" t-att-title='groups_tooltip'>Discard &amp; Edit in backend</a>
```

Comment:
@JKE-be @fja-odoo Do you still remember the logic behind this hardcoded id ?  Seems strange to me, but there might be some logic I'm not aware of :D.

Reply:
seems a really bad idea, so I will say, an error missed during the review.
Or temp fix in local commited by error...

Reply:
@rdeodoo seems louch, maybe could you check and fix it? ;)

Reply:
"Seems louch" or "seems louche" ?

Reply:
Indeed, was a quick one, done at https://github.com/odoo/odoo/pull/86898/files